org-owners-guide.md

```diff
@@ -0,0 +1,66 @@
+# GitHub Organization Owners Guide
+
+The Kubernetes project leverages multiple GitHub organizations to store and
+organize code. This guide contains the details on how to run those organizations
+to for CNCF compliance and for the guidelines of the community.
+
+## Organization Naming
+
+Kubernetes managed organizations should be in the form of `kubernetes-[thing]`.
+For example, [kubernetes-client](https://github.com/kubernetes-client) where the
+API clients are housed.
+
+Prior to creating an organization please contact the steering committee for
+direction and approval.
+
+Note: The CNCF, as part of the Linux Foundation, holds the trademark on the
+Kubernetes name. All GitHub organizations with Kubernetes in the name should be
+managed by the Kubernetes project or use a different name.
```

Comment:
^ is that something we can actually enforce? I'm not pushing back here, just curious.

Reply:
@michelleN if the CNCF doesn't enforce the trademark on the name than it can loose the ability to do this. It's a general thing that you need to enforce usage on a name or you loose the right to. It's legal and as I'm not a lawyer I don't really know the details. I'm sure the Linux Foundation and CNCF folks know what to do, though.

Because the CNCF owns the name there are channels through GitHub to deal with this if it were to come to that (which it usually doesn't).